src/ui/molecules/stepper/stepper.scss

```diff
@@ -86,11 +86,11 @@ $colors: (
     overflow-y: auto;
     width: 100%;
     padding: 2px;
+  }
 
-    &.error {
-      @include with-theme() {
-        border: solid themed(error);
-      }
+  &.error {
```

Comment:
I would have just moved the `error` classname above `.okp4-stepper-step-content` because it still refers to `.okp4-stepper-body`